shelly/src/main/java/au/com/jtribe/shelly/SocialShare.java

```diff
@@ -101,6 +104,8 @@ public SocialShare video(Uri videoUri) {
     return this;
   }
```

Comment:
can you add `@CheckResult` and `@NonNull` to the rest of the methods? 

 * CheckResult shows a warning if the calling code uses the method but doesn't use the result.
 * NonNull declares things that cannot be null so that the IDE can provide a hint to the programmer (should still check manually)
 * Nullable is the opposite of NonNull.

In Shelly we should annotate everything possible with these annotations so that the IDE can hint the programmers to do the right thing.